### PR TITLE
coerage

### PR DESCRIPTION
**Pull request by HarrisChu**

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -1,7 +1,5 @@
 <?xml version="1.0" encoding="UTF-8"?>
-<project xmlns="http://maven.apache.org/POM/4.0.0"
-         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
-         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
```
Comment (Nicole00) [inline pom.xml:2]: split into different lines.

File: pom.xml [hunk 2]
```diff
@@ -77,6 +75,25 @@
...
+                        <goals>
+                            <goal>report</goal>
+                        </goals>
+                    </execution>
+                </executions>
+            </plugin>
         </plugins>
     </build>
-</project>
+</project>
```
Comment (Nicole00) [inline pom.xml:99]: add extra line at end

File: pom.xml (hunk 1 above)
Comment (HarrisChu) [reply pom.xml:2]: formate by vscode.

refer https://github.com/apache/dubbo/blob/3.0/pom.xml#L17
https://github.com/apache/kylin/blob/main/pom.xml#L19

File: pom.xml (hunk 2 above)
Comment (HarrisChu) [reply pom.xml:99]: still auto format
refer https://github.com/apache/dubbo/blob/3.0/pom.xml#L755
https://github.com/apache/kylin/blob/main/pom.xml#L2143

File: pom.xml (hunk 1 above)
Comment (Nicole00) [reply pom.xml:2]: should keep the same  with nebula's all java project. refer https://github.com/neo4j/neo4j-java-driver/blob/3eb361338b930d26becb4c23e961661295c26a4e/pom.xml#L1

**Comment by Nicole00** [comment]
add coverage after the repo is split.

**Pull request closed without merging**